src/dotnet/Chat.Service/ChatsBackend.cs

```diff
@@ -708,6 +714,43 @@ await dbContext.ChatEntries
         context.Operation().Items.Set(chatEntriesToInvalidate);
     }
 
+    public virtual async Task OnCreateNotesChat(ChatsBackend_CreateNotesChat command, CancellationToken cancellationToken)
+    {
+        if (Computed.IsInvalidating())
+            return; // It just spawns other commands, so nothing to do here
+
+        var userId = command.UserId;
+        var dbContext = await CreateCommandDbContext(cancellationToken).ConfigureAwait(false);
+        await using var __ = dbContext.ConfigureAwait(false);
+
+        var hasNotesChat = await dbContext.Chats
+            .Join(dbContext.Authors, c => c.Id, a => a.ChatId, (c, a) => new { c, a })
+            .AnyAsync(x => x.a.UserId == userId && x.c.Tag == Constants.Chat.Tags.Notes, cancellationToken)
+            .ConfigureAwait(false);
+
+        if (hasNotesChat)
+            return;
+
+        var createNotesCommand = new ChatsBackend_Change(
+            ChatId.None,
+            null,
+            new Change<ChatDiff> {
+                Create = new ChatDiff {
+                    Title = "Notes",
+                    Kind = ChatKind.Group,
+                    IsPublic = false,
+                    IsTemplate = false,
+                    AllowGuestAuthors = false,
+                    AllowAnonymousAuthors = false,
+                    AllowSingleAuthorOnly = true,
```

Comment:
Do we really need to persist AllowSingleAuthorOnly property? May be we can rely on Tag? Certain tags will indicate that it's a SingleAuthorOnly chat.

Reply:
As we already use flags to evaluate permissions it makes sense to use same approach for new restriction

Reply:
IMO it's not a good idea to do it this way too.

Here is what I would do:
1. Rename `Tag` field to `SystemTag` (`Symbol`)
2. Rename `AllowSingleAuthorOnly` property to `HasSingleAuthor()`, which uses `SystemTag` to compute its value.